### DISCUSSION TITLE
Uncaught ReferenceError while obtaining token

### DISCUSSION THREAD

**Discussion by SiPhung953** [marioparaschiv/nitro-sniper#286] [Q&A]
![image](https://user-images.githubusercontent.com/93171516/138797233-c9b91150-f9ce-4779-832b-44cb1660dcb2.png)
For some reasons, I can't seem to obtain my Token anymore. I tried closing Discord and restart, open the web version, still doesn't work.
I copied the code from the Youtube video provided here.
Did Discord updated something so that we can't obtain our Token, or is it just me?

**Comment by WSrujan** [comment]
Yes discord changed major part in console there will be updated codes in GitHub you just have to find and be careful to enter correct code

**Reply by WSrujan** [reply]
Look for console updated 2 days old
